recipe/meta.yaml

```diff
@@ -17,7 +17,7 @@ host:
   - pip
```

Comment:
This host section can be removed. The build happens in the output.

Reply:
Done